src/gui_common/CustomRichTextLabel.cs

```diff
@@ -124,7 +134,12 @@ private void ParseCustomTags()
             return;
         }
 
+        var old = rawTranslatedMarkup;
         var translated = TranslationServer.Translate(extendedBbcode);
+        if (translated == old)
+            return;
```

Comment:
```suggestion
{
   reactToLanguageChange = false;
}
else
{
    reactToLanguageChange = true;
}
```
And then modifying the notification to only trigger the update if that bool is true, is what's basically my suggestion on how to handle this.

Reply:
Alright, my last commit should be closer to your suggestion.